### PR TITLE
[CAT-1425] : Removing RedHat/Scientific/OracleLinux 6

### PR DESCRIPTION
**Pull request by rajat-puppet**
## Summary
Removing EOL RedHat/Scientific/OracleLinux 6.

## Additional Context
Add any additional context about the problem here. 
- [ ] Root cause and the steps to reproduce. (If applicable)
- [ ] Thought process behind the implementation.

## Related Issues (if any)
Mention any related issues or pull requests.

## Checklist
- [ ] 🟢 Spec tests.
- [ ] 🟢 Acceptance tests.
- [ ] Manually verified. (For example `puppet apply`)

### CODE REVIEW

File: spec/spec_helper_acceptance_local.rb [hunk 1]
```diff
@@ -62,7 +62,7 @@ def fetch_os_name
   c.before :suite do
     # Depmod is not availible by default on our AlmaLinux/CentOS 8 docker image
     LitmusHelper.instance.run_shell('yum install kmod -y') if ['almalinux-8', 'centos-8'].include?("#{fetch_os_name}-#{os[:release].to_i}")
-    if ['centos-6', 'centos-7', 'oraclelinux-6', 'scientific-6', 'scientific-7'].include?("#{fetch_os_name}-#{os[:release].to_i}")
```
Comment (Ramesh7) [inline spec/spec_helper_acceptance_local.rb:65]: we should be cleaning up centos-6.

File: spec/acceptance/firewall_attributes_exceptions_spec.rb [hunk 2]
```diff
@@ -19,9 +19,8 @@
 
   # --bytecode is only supported by operatingsystems using nftables (in general Linux kernel 3.13, RedHat 7 (and derivates) with 3.10)
   # Skipping those from which we know they would fail.
-  describe 'bytecode property', unless: (os[:family] == 'redhat' && os[:release][0] <= '6') ||
```
Comment (Ramesh7) [inline spec/acceptance/firewall_attributes_exceptions_spec.rb:22]: we should be also good to remove :
```
(os[:family] == 'sles' && os[:release][0..1] <= '11')
```
as we don't have 11, we only support 12 & 15.

File: spec/acceptance/firewall_attributes_exceptions_spec.rb [hunk 3]
```diff
@@ -19,9 +19,8 @@
...
-                                        (os[:family] == 'sles' && os[:release][0..1] <= '11') ||
-                                        (fetch_os_name == 'oraclelinux' && os[:release][0] <= '7') ||
```
Comment (Ramesh7) [inline spec/acceptance/firewall_attributes_exceptions_spec.rb:24]: may be we keep oraclelinux condition as it is, as we have only 1 version i.e. 7, once we add new version that time we can evaluate it?

File: spec/acceptance/firewall_attributes_ipv6_exceptions_spec.rb [hunk 4]
```diff
@@ -86,29 +86,6 @@ class { '::firewall': }
...
-              expect(r.stderr).to match(%r{Multiple #{type} elements are available from iptables version})
-            end
-          end
-
-          it 'does not contain the rule' do
-            run_shell('ip6tables-save') do |r|
-              expect(r.stdout).not_to match(%r{-A INPUT -p (tcp|6) -m addrtype --#{type.tr('_', '-')} LOCAL -m addrtype ! --#{type.tr('_', '-')} LOCAL -m comment --comment "616 - test" -j ACCEPT})
-            end
-          end
-        end
```
Comment (Ramesh7) [inline spec/acceptance/firewall_attributes_ipv6_exceptions_spec.rb:111]: at line 120 the line seems to be problematic :
```
describe 'ipset', if: (os[:family] == 'redhat' && os[:release].start_with?('14')) do
```
it should be I think : 
```
describe 'ipset', if: (os[:family] == 'debian' && os[:release].start_with?('14')) do
```

File: spec/acceptance/firewall_attributes_exceptions_spec.rb (hunk 2 above)
Comment (rajat-puppet) [reply spec/acceptance/firewall_attributes_exceptions_spec.rb:22]: Done.

File: spec/acceptance/firewall_attributes_exceptions_spec.rb (hunk 3 above)
Comment (rajat-puppet) [reply spec/acceptance/firewall_attributes_exceptions_spec.rb:24]: Yes, we'll keep it as is.

File: spec/acceptance/firewall_attributes_ipv6_exceptions_spec.rb (hunk 4 above)
Comment (rajat-puppet) [reply spec/acceptance/firewall_attributes_ipv6_exceptions_spec.rb:111]: Added debian.

File: spec/spec_helper_acceptance_local.rb (hunk 1 above)
Comment (rajat-puppet) [reply spec/spec_helper_acceptance_local.rb:65]: Removed centos-6.

**Review by Ramesh7 — APPROVED** [review]

**Pull request merged**